contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -673,8 +699,11 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
      *      parameters.
      *      This function works whether the strategy contract has liquidity
      *      position in the pool or not.
+     * @param dryRun  when this property is set to true the function doesn't throw
+     *        an exception but returns a bool. `True` means price within expected
+     *        range and `false` means outside that range.
      */
-    function _checkForExpectedPoolPrice() internal {
+    function _checkForExpectedPoolPrice(bool dryRun) internal returns (bool) {
```

Comment:
⚪ I think this boolean would be better flipped. I think it's more expected when a function flag enables features. True = bigger thing happens. In this case reverting is a much bigger and more impactful action than returning a value.

Reply:
Yes good point. And aside from what you've already pointed out `throwException` is clearer parameter name than `dryRun`

Reply:
done here: https://github.com/OriginProtocol/origin-dollar/pull/2266/commits/c7346e63a48e9658dc1973327ff93e3b7f959998